types/artifacts.d.ts

```diff
@@ -240,6 +240,8 @@ declare global {
         crossOrigin: string | null
         /** Where the link was found, either in the DOM or in the headers of the main document */
         source: 'head'|'body'|'headers'
+        /** Path that uniquely identifies the node in the DOM */
```

Comment:
can you add to the comment that this won't be defined when `source` is `'headers'` (would have helped me :)

Reply:
Good point! Done